src/components/Search.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { allCountries } from '../redux/categorySlice';
+import '../css/search.css';
+
+const Search = () => {
```

Comment:
- [ ] I can't to see any 'see more ' botton or click on the country name to open the details , after searching in search bar. kindly add it there also. And after clicking it will open details in  url like "/countries/Algeria" or "/countries/:id".

![Screenshot from 2023-07-01 19-35-49](https://github.com/mosidrum/react_capstone/assets/26767383/962145eb-9a94-4755-ae9d-202971df04bb)

** Using the same component to show these details will be easy way.